cypress/fixtures/sportsArticlesResponse.json

```diff
@@ -3,37 +3,32 @@
     {
       "id": 482,
       "title": "Ronaldo and Messi",
-      "body": "Champions of whatever",
-      "created_at": "2021-12-13T10:24:38.354Z",
-      "updated_at": "2022-01-13T10:24:38.354Z"
+      "teaser": "Champions of whatever",
+      "published": "2021-12-13T10:24:38.354Z"
```

Comment:
This fixture file doesn't match the api response.
We return the category as well.
Example:
{"id"=>236,
    "title"=>"Covid-19 is a pandemic",
    "teaser"=>"This is my article body...",
    "published"=>"2022-01-23T16:32:50.881Z",
    "category"=>"Politics"}